spec/requests/withdraw_spec.rb

```diff
@@ -21,6 +21,16 @@
       )
     end
 
+    it "returns an error when publishing-api is down" do
+      publishing_api_isnt_available
+      login_as(managing_editor)
+
+      post withdraw_path(published_edition.document), params: { public_explanation: "Just cos" }
+      follow_redirect!
+
+      expect(response.body).to include(I18n.t!("withdraw.new.flashes.publishing_api_error.title"))
```

Comment:
Are we missing a status code assertion here?

```ruby
      expect(response.status).to eq(502)
```

Reply:
Hmm not really because the request is following a redirect so before the redirect it will be a `302` and after the redirect the response status will be `200` (successfully rendered error page) .